boa/vyper/contract.py

```diff
@@ -465,15 +466,17 @@ def at(self, *args, **kwargs):
     def ast_map(self):
         return ast_map_of(self.compiler_data.vyper_module)
 
+    def _get_fn_from_computation(self, computation):
+        node = self.find_source_of(computation.code)
+        if isinstance(node, vy_ast.FunctionDef):
```

Comment:
i think this breaks the previous None check?

Reply:
It will not, since that is in the profiling section https://github.com/vyperlang/titanoboa/blob/dae534980793c36704f97f53bdecc28b8bb591ee/boa/profiling.py#L181-L188